tests/test_broker_config.cpp

```diff
@@ -17,12 +17,21 @@
  * https://github.com/ETCLabs/RDMnetBroker
  *****************************************************************************/
 
+#define NOMINMAX
```

Comment:
I need to remember to do this more.

Reply:
Yeah, I only did it because it was breaking my compile...